src/utils/create-archive.ts

```diff
@@ -150,15 +150,24 @@ export default async function createArchive(
     return false;
   };
 
-  const fileList: string[] = fs.readdirSync(projectPath).filter(ignoreFN);
+  const addPrefix = (input: string): string => {
+    if (input.startsWith('@')) {
+      return `./${input}`;
+    }
+    return input;
+  };
+
+  const fileList: string[] = fs
+    .readdirSync(projectPath)
+    .filter(ignoreFN)
+    .map(addPrefix);
 
-  return await tar.create(
+  return tar.create(
     {
       gzip: {
         level: 9,
       },
       cwd: projectPath,
-      filter: ignoreFN,
```

Comment:
`readdir` only works on the root level, but `tar` traverses internal directories as well. So we should keep this line.